### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#49] [Typos （English）]
1-20 characters I think is wrong, it should not be there, it should start from 1-3 characters. 
![Uploading IMG_20220115_155547.jpg…]()

**Comment by Abdullahoda** [comment]
![IMG_20220115_155547](https://user-images.githubusercontent.com/97207588/149622848-0aa479d3-56c1-4e3b-9e0a-88696dac7356.jpg)

**Comment by HalloKyle** [comment]
We will make a new description for this page thank you for your support.
